CONTRIBUTING.md

```diff
@@ -215,6 +215,11 @@ Per the
 [API guidelines](https://rust-lang.github.io/api-guidelines/naming.html#getter-names-follow-rust-convention-c-getter),
 `get_()` prefixes are discouraged.
 
+#### Alphabetize enum variants
+
+When implementing or modifying an `enum` type, list its variants in alphabetical
```

Comment:
Unless overriding reasons not to, for example a discriminant order imposed by a standard.

Reply:
Added a caveat about this.